packages/ckbtc/src/utils/btc.utils.ts

```diff
@@ -214,3 +216,14 @@ export const parseBtcAddress = ({
       throw new ParseBtcAddressUnsupportedAddressTypeError();
   }
 };
+
+/**
+ * Parse a Bitcoin network.
+ *
+ * Parse the Bitcoin network given as a string into a network object of type `{ network: null }`, that is used in the Bitcoin canister.
+ *
+ * @param {BitcoinNetwork} network The Bitcoin network to parse
+ * @returns {network} The parsed network
+ */
+export const parseBitcoinNetwork = (network: BitcoinNetwork): network =>
```

Comment:
I don't think we should expose this function to consumer.

Moreover it's a bit misleadint that it is called `parse...` but does not do much except mapping particularly given that the same moduel contains `parseBtcAddress`.

In summary: I think it should be renamed and moved.

Reply:
right! mapping sounds better

about exposing: agreed! I am actually thinking that we would need it at some point; so I will do a separate PR, when the need comes